org.multimc.MultiMC.yml

```diff
@@ -0,0 +1,51 @@
+app-id: org.multimc.MultiMC
+default-branch: stable
+runtime: org.kde.Platform
+runtime-version: '5.15'
+command: multimc
+separate-locales: false
+sdk: org.kde.Sdk
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --share=ipc
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+build-options:
+  append-path: /usr/lib/sdk/openjdk11/bin
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+modules:
+  - name: openjdk
```

Comment:
I'd consider bundling regular org.freedesktop.Sdk.Extension.openjdk just for the sake of it, as I assume it's something people would like to have.

Reply:
This wasn't resolved

Reply:
It was? There are 3 JDK versions now.